kahuna/public/stylesheets/main.css

```diff
@@ -518,6 +518,15 @@ input.search-query__input {
     display: none;
 }
 
+.file-uploader__select-files {
+    padding: 5px 10px;
+}
+
+@media screen and (max-width: 1200px) {
```

Comment:
Whilst we've not got a pre-processor, should there be a separate file for responsive rules so we wouldn't have to repeat this query? i.e

```css
@media screen and (max-width: 1200px) {
    body: {
        color: hotpink;
   }

/* more rules that use media queries */

   .file-uploader__select-files--label {
     display: none; 
   }
}
```

vs.

```css
@media screen and (max-width: 1200px) {
    body: {
        color: hotpink;
   }
  }

/* more rules that may or may not use media queries */

@media screen and (max-width: 1200px) {
   .file-uploader__select-files--label {
     display: none; 
   }
}

```